src/django/api/views/v1/utils.py

```diff
@@ -108,3 +109,26 @@ def _wrapped_view(self, request, *args, **kwargs):
         except OpenSearchServiceException as e:
             return handle_opensearch_exception(e)
     return _wrapped_view
+
+
+def is_valid_uuid(uuid_string):
```

Comment:
This code is already present in `src/django/api/serializers/v1/opensearch_common_validators/moderation_id_validator.py`. Could you use this util function there as well?